README.md

```diff
@@ -1,11 +1,55 @@
-# Your GitHub Learning Lab Repository for Community Starter Kit
+# welcome bot: A Probot App
 
-Welcome to **your** repository for your GitHub Learning Lab course. This repository will be used during the different activities that I will be guiding you through to learn about making your project friendlier for collaboration.
+DESCRIPTION PLACEHOLDER
```

Comment:
What the project does
    The features and benefits of the project
    How users can get started with the project
    Where users can get help with your project
    Who maintains and contributes to the project